packages/shared/components/molecules/AssetList.svelte

```diff
@@ -1,12 +1,23 @@
 <script lang="typescript">
     import { localize } from '@core/i18n'
-    import { assetFilter, IAccountAssets } from '@core/wallet'
+    import { assetFilter, IAccountAssets, IAsset } from '@core/wallet'
     import { openPopup } from '@lib/popup'
+    import VirtualList from '@sveltejs/svelte-virtual-list'
     import { AssetTile, Text, Filter } from 'shared/components'
     import { TextType } from 'shared/components/Text.svelte'
 
     export let assets: IAccountAssets
 
+    let assetList: IAsset[]
+    $: {
+        const list = []
+        if (assets?.baseCoin) {
+            list.push(assets.baseCoin)
+        }
+        list.push(...assets?.nativeTokens)
+        assetList = list
+    }
```

Comment:
Will this have any side effects of loading the whole list of tokens whenever the assets change?

Reply:
What do you mean by side effects?